vmngclient/api/templates.py

```diff
@@ -242,8 +291,9 @@ def load_running(self, device: Device) -> None:
             device (Device): The device from which load config.
         """
         endpoint = f"/dataservice/template/config/running/{device.uuid}"
-        config = cast(dict, self.session.get_json(endpoint))
+        config = self.session.get_json(endpoint)
         self.config = CiscoConfParse(config['config'].splitlines())
+        logger.warning(f"Template loaded from {device.hostname}.")
```

Comment:
why are warnings everywhere ? All logged warnings in this module looks like regular operations

Reply:
yes you are right - corrected